### ISSUE TITLE
money() displays ¤ instead of $.

### ISSUE THREAD

**Issue by lovecoding-git** [calebporzio/awesome-helpers#36]
I tried `money(35.55)` => This showed me` ¤35.88`.

Why this issue?

**Comment by adrianb93** [comment]
The tests _(which seem to be passing)_ are passing in `'en_US.utf-8'` as the locale to get the dollar symbol.

This should work:

```
money(35.55, true, 'en_US.utf-8');
```

**Comment by adrianb93** [comment]
Putting this in your AppServiceProvider might also work:

```
locale_set_default('en_US.utf-8');
```

**Comment by lovecoding-git** [comment]
> This should work:

> money(35.55, true, 'en_US.utf-8');

Same issue for this.

![image](https://user-images.githubusercontent.com/61916777/84593736-35ce9780-ae80-11ea-9060-b369f9467902.png)

**Comment by lovecoding-git** [comment]
> locale_set_default('en_US.utf-8');

same issue for this too.

**Comment by adrianb93** [comment]
Sorry I couldn't be of more help.

I was able to reproduce your issue on a Forge server, but the following had worked for me in tinker:

```php
function money($input, $showCents = true, $locale = null)
{
    setlocale(LC_MONETARY, $locale ?: locale_get_default());
    
    $numberOfDecimalPlaces = $showCents ? 2 : 0;

    $formatter = numfmt_create('en_US', \NumberFormatter::CURRENCY);
    $formatter->setAttribute(\NumberFormatter::FRACTION_DIGITS, $numberOfDecimalPlaces);

    return numfmt_format_currency($formatter, $input, trim(localeconv()['int_curr_symbol']));
}

money(35.88)
=> "¤35.88"

money(35.88, true, 'en_US.utf-8')
=> "$35.88"

locale_set_default('en_US.utf-8');
money(35.88)
=> "$35.88"
```

When you solve your problem, please leave a comment in case it helps anyone with the same issue 🙂

**Comment by lovecoding-git** [comment]
![image](https://user-images.githubusercontent.com/61916777/84626552-d9b75200-af17-11ea-93d4-495ba6ba957f.png)

This is my Tinker result.

**Comment by dougfelton** [comment]
TL;DR
I'm on Windows and putting this in AppServiceProvider solved the issue for me:

```php
// fix issue with calebporzio/awesome-helpers money() function on Windows dev machine
if ($this->app->isLocal()) {
    locale_set_default('english-us');
}
```

Please bear in mind that I have no idea what I'm talking about and may be mangling the idea:

The problem seems to stem from how Windows handles setlocale().  It appears that on Windows, it sets the locale names "from the system's regional/language settings (accessible via Control Panel)" (from setlocale() man page).  In money.php in the setlocale() call, the locale_get_default() call returns 'en_US.utf-8' but Windows seems to need an equivalent language string instead.  Those equivalents are listed on this page and is where I pulled 'english-us' from.

https://docs.microsoft.com/en-us/cpp/c-runtime-library/language-strings?view=vs-2019

**Comment by dougfelton** [comment]
Updating my last comment regarding a workaround I found.  I added some code to detect when the OS is Windows:

```
if ($this->app->isLocal()) {
    if (strcasecmp(substr(PHP_OS, 0, 3), 'WIN') == 0) {
        locale_set_default('english-us');
    } else {
        locale_set_default('en_US');
    }
}
```

**Comment by nullthoughts** [comment]
Silly thought, but shouldn't the money method be using `app()->getLocale()` instead of `locale_get_default()` ? 

This way it would utilize the locale set in config/app.php

Edit: Forked, tested and PR opened (PR #40)
